template/package.json

```diff
@@ -6,30 +6,29 @@
     "dev": "meta dev",
     "build": "meta build",
     "lint": "npm run lint:scripts && npm run lint:styles",
-    "lint:scripts": "eslint web/wp-content/themes/<%= slug %>/src/js --ext=.js,.vue",
-    "lint:styles": "stylelint 'web/wp-content/themes/<%= slug %>/src/**/*.(scss|vue)'"
+    "lint:scripts": "eslint web/wp-content/themes/test2/src/js --ext=.js,.vue",
+    "lint:styles": "stylelint 'web/wp-content/themes/test2/src/**/*.(scss|vue)'"
   },
   "author": "Studio Meta <agence@studiometa.fr> (https://www.studiometa.fr)",
   "license": "MIT",
   "devDependencies": {
-    "@babel/plugin-transform-runtime": "^7.10.5",
-    "@babel/preset-env": "^7.10.2",
+    "@babel/plugin-transform-runtime": "^7.12.1",
+    "@babel/preset-env": "^7.12.1",
     "@fullhuman/postcss-purgecss": "^2.2.0",
     "@studiometa/browserslist-config": "^1.0.1",
-    "@studiometa/eslint-config": "^2.1.2",
+    "@studiometa/eslint-config": "^2.1.3",
     "@studiometa/prettier-config": "^2.0.1",
     "@studiometa/stylelint-config": "^2.0.0",
     "@studiometa/tailwind-config": "^1.0.2",
-    "@studiometa/webpack-config": "^2.0.1",
+    "@studiometa/webpack-config": "^2.4.1",
     "autoprefixer": "^9.8.0",
```

Comment:
Same as Purge CSS, the `autoprefixer` and `cssnano` packages are included in the Webpack configuration, we can remove them here.